client/my-sites/marketplace/components/reviews-summary/index.tsx

```diff
@@ -68,7 +68,7 @@ export const ReviewsSummary = ( { slug, productName, productType }: Props ) => {
 				productType={ productType }
 			/>
 			<div className="reviews-summary__container">
-				{ numberOfReviews !== null && (
+				{ numberOfReviews !== null && numberOfReviews >= 3 && (
```

Comment:
Worth a comment explaining why? "Only display if we have reached a minimum sample size to reduce outliers unfairly impacting the score"

and/or using a well-named const